lib/auth/auth.go

```diff
@@ -384,10 +384,18 @@ var (
 		[]string{teleport.TagVersion},
 	)
 
+	migrations = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
```

Comment:
Add a `Namespace: teleport.MetricNamespace` to this (and `teleport_` to metrics.mdx).

Reply:
Done in 21b2303ccdf26d0204203d41964d2430171a0055.